tests/end2end/test_nodecellar.py

```diff
@@ -39,4 +39,4 @@ def _verify_deployed_service_in_storage(service_name, model_storage):
     service = service_templates[0].services[service_name]
     assert service.name == service_name
     assert len(service.executions) == 0  # dry executions leave no traces
-    assert len(service.nodes) == 10
+    assert len(service.nodes) == 13
```

Comment:
The policy is on the `node_cellar_group` which contains three 3 templates. So we get 3 extra nodes.